src/fee_lab_to_nwb/scherrer_ophys/scherrerophysimagingextractor.py

```diff
@@ -1,15 +1,35 @@
-from typing import Tuple
+from typing import Tuple, Optional
 
 import numpy as np
-from neuroconv.datainterfaces.behavior.movie.movie_utils import (
-    VideoCaptureContext,
-    HAVE_OPENCV,
-    INSTALL_MESSAGE,
-)
+from neuroconv.datainterfaces.behavior.movie.movie_utils import VideoCaptureContext
 from roiextractors import ImagingExtractor
 from neuroconv.utils import FilePathType, ArrayType
 from roiextractors.extraction_tools import NumpyArray
 
+try:
+    import cv2
+
+    HAVE_OPENCV = True
+except ImportError:
+    HAVE_OPENCV = False
+INSTALL_MESSAGE = "Please install opencv to use the VideoCaptureContext class! (pip install opencv-python)"
```

Comment:
You can go ahead and make opencv a hard requirment of the installation.

Actually, it already is: https://github.com/catalystneuro/fee-lab-to-nwb/blob/main/requirements.txt#L4

So you can just import it here without lazy try/except

Reply:
Thanks for catching this!